docs/developer/architecture/centralized-cli.rst

```diff
@@ -0,0 +1,57 @@
+Centralized CLI
+===============
+
+Context
+-------
+
+MetalK8s comes with a set of services to operate and monitor the K8s cluster.
+All operations that need to be performed by the Platform Administrator could be
+categorized as follow:
+- Cluster Resources Administration (Nodes, Volumes, Deployments, ...)
+- Cluster Administration (Install, Upgrade, Downgrade, Backup, Restore, ...)
+- Solution Administration (CRUD Environment, Import/Remove Solution, ...)
+- Cluster Service Administration (Configure Dex, Prometheus, Alert Manager, ...)
+
+K8s provides the kubctl CLI, which is distributed and enables all kind of
+interaction with all resources available in etcd DB but its usage often requires
+to build verbose yaml files. Also kubectl tool only expose the standard K8s
+resources but not the custom ones, such as Volumes.
+
+Currently, MetalK8s provides other set of CLI or manual procedures, but those
```

Comment:
`a set of tools` not CLI.

I don't think CLI means what you think. A CLI is defined as `A user interface in which the user interacts with the computer through lines of text instead of graphical images.`

What we have is a bunch of script that we can run, and they are not really interactive.
You could say that this bunch of scripts defines a CLI for MetalK8s, but a script by itself isn't a CLI.

Reply:
done